packages/data-context/src/actions/WizardActions.ts

```diff
@@ -83,44 +88,44 @@ export class WizardActions {
 
   /// reset wizard status, useful for when changing to a new project
   resetWizard () {
-    this.data.chosenBundler = null
-    this.data.chosenFramework = null
-    this.data.chosenLanguage = 'js'
+    this.ctx.update((coreData) => {
+      coreData.wizard.chosenBundler = null
+      coreData.wizard.chosenFramework = null
+      coreData.wizard.chosenLanguage = 'js'
+      coreData.wizard.detectedBundler = null
+      coreData.wizard.detectedFramework = null
+    })
 
-    return this.data
+    return this.ctx.coreData.wizard
   }
 
   async initialize () {
     if (!this.ctx.currentProject) {
       return
     }
 
-    this.ctx.update((coreData) => {
-      coreData.wizard.detectedFramework = null
-      coreData.wizard.detectedBundler = null
-      coreData.wizard.detectedLanguage = null
-    })
+    this.resetWizard()
 
     await this.detectLanguage()
     debug('detectedLanguage %s', this.data.detectedLanguage)
     this.data.chosenLanguage = this.data.detectedLanguage || 'js'
 
     try {
-      const detected = detect(await fs.readJson(path.join(this.ctx.currentProject, 'package.json')))
+      const detected = detect(this.ctx.currentProject)
```

Comment:
Can we remove the try/catch now that we're not awaiting? Nothing else seems like it should (reasonably) throw.

Reply:
Done